app/services/credit_notes/refunds/gocardless_service.rb

```diff
@@ -96,7 +96,6 @@ def create_gocardless_refund
             total_amount_confirmation: credit_note.refund_amount_cents,
             links: {payment: payment.provider_payment_id},
             metadata: {
```

Comment:
@vincent-pochet We can add a comment with an explanation to not add another (fourth) key in the future.